backendStub/service/FrikkService.js

```diff
@@ -0,0 +1,237 @@
+'use strict';
+
+
+/**
+ * Return data of a specific menu
+ * Returns all the data about the menu with given menuID, even the menu items inside it. The photos of the menu items are saved in the Amazon s3 storage, so the links to the cloud storage are also returned. The frontend will directly download them from the cloud storage, they won't be sent by the backend with this endpoint.
+ *
+ * menuID Integer ID of the menu to be returned
+ * returns MenuRetrievedToCustomer
+ **/
+exports.getMenuCustomer = function(menuID) {
+  return new Promise(function(resolve, reject) {
+    var examples = {};
+    examples['application/json'] = {
+  "restaurant" : {
+    "restaurantName" : "Emilio's Pizza",
+    "address" : "André route 45",
+    "city" : "Rome",
+    "country" : "Italy",
+    "latitude" : 48.4281350802915,
+    "longitude" : -110.0792542197085
+  },
+  "menuName" : "Emilio's menu of the day",
+  "description" : "Our special menu of today",
+  "tags" : [ {
+    "name" : "Italian",
+    "color" : "#FFBC8C"
+  }, {
+    "name" : "Mediterranean",
+    "color" : "#FFBC8C"
+  } ],
+  "menuRating" : 5,
+  "menuItems" : [ {
+    "name" : "Spaghetti alla carbonara",
+    "description" : "Fantastic italian dish made of spaghetti, pig cheek, eggs, black pepper, pecorino romano",
+    "type" : "dish",
+    "priceEuros" : 10,
+    "tags" : [ {
+      "name" : "Mediterranean",
+      "color" : "#FFBC8C"
+    }, {
+      "name" : "Pasta",
+      "color" : "#99C99B"
+    }, {
+      "name" : "Italian",
+      "color" : "#FFBC8C"
+    } ],
+    "imageLink" : "www.cloudStorage.com/Carbonara"
+  }, {
+    "name" : "Polpette al sugo",
+    "description" : "Meatballs with tomato sauce",
+    "type" : "dish",
+    "priceEuros" : 7,
+    "tags" : [ {
+      "name" : "Mediterranean",
+      "color" : "#FFBC8C"
+    }, {
+      "name" : "Meat",
+      "color" : "#FFBC8C"
+    }, {
+      "name" : "Italian",
+      "color" : "#FFBC8C"
+    } ],
+    "imageLink" : "www.cloudStorage.com/Meatballs"
+  } ]
+};
+    if (Object.keys(examples).length > 0) {
+      resolve(examples[Object.keys(examples)[0]]);
+    } else {
+      resolve();
+    }
+  });
+}
+
+
+/**
+ * Return all the menus of the restaurant
+ * Return all the menus of the restaurant. Since authentication is required, the backend is able to get which restaurant is involved from the authentication token.
+ *
+ * returns List
+ **/
+exports.getOwnMenus = function() {
+  return new Promise(function(resolve, reject) {
+    var examples = {};
+    examples['application/json'] = [ {
+  "menuID" : 2,
```

Comment:
The two menus returned here have the same ID, can this be changed? It violates the unique item constraints of menus